docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:latest
```

Comment:
We should somehow differentiate in the name that this file is for Full and not Light node. Eventually, we should make one for Light,